GX-TimeFrequency/src/minimax_tau.F90

```diff
@@ -2921,10 +2925,24 @@ subroutine get_points_weights_tau(grid_size, e_range, ac_we, ierr)
        allocate(aw%aw_erange_matrix(2*grid_size, bup+1))
        call set_aw_array(grid_size, aw)
 
+
        ! Select energy region with binary search
        ien = find_erange(bup, aw%energy_range, e_range)
+
+       ! Scale grids for large sizes when erange falls in the
+       ! first energy range
+       e_ratio = 1.0_dp
+       if (ien == 1 .and. grid_size > 20) then
+          e_ratio = aw%energy_range(1)/e_range
+          if (e_ratio > 1.5_dp) then
```

Comment:
1.5_dp can be a named parameter. And again, any comment/reference on why 1.5 would be very appreciated